index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,700;1,300;1,500&display=swap" rel="stylesheet">
+  <!-- Material Icons -->
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+  
+  <main class="main">
+ 
+<header class="nav-bar">    
+  <div class="logo-left">My Logo</div>
+ <i class="material-icons">menu</i>
+</header>
+
+<section class="headline">
+   <h1>I'm Cookey<br>Glad to see <br> you!</h1>
+    <p>I'm a software developer! I can help you build a product, feature or website. Look through some experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
+    </p>
+
+
+    <h3>LET'S CONNECT</h3>
+
+  <div class="contacts">
+   
+    <a href="">
+      <img src="./images/twitter.svg" width="20"  height="15" alt="twitter">
+    </a>
+    <a href="">
+      <img src="./images/linkedin.svg" width="20"  height="15" alt="twitter">
+    </a>
+    <a href="">
+      <img src="./images/medium.svg" width="20"  height="15" alt="twitter">
+    </a>
+    <a href="">
+      <img src="./images/github.svg" width="20"  height="15" alt="twitter">
+    </a>
+    <a href="">
+      <img src="./images/Vector.svg" width="20"  height="15" alt="twitter">
+    </a>
```

Comment:
- [ ] Your h3 tag is not the same as it has a different color and it has no font family and your social- icons have no spacing, make a gap in between the icons and it is bigger has to be bigger in size.
### your design
![his last](https://user-images.githubusercontent.com/81727988/170712323-8fa5aa01-a33d-46f3-a340-2fd1e1157c60.png)
### Figma design
![last](https://user-images.githubusercontent.com/81727988/170712385-1c397a2f-8b1a-4bf1-9cc9-0dc1d608548e.png)